lib/fortnox/api.rb

```diff
@@ -26,22 +26,11 @@ module API
     }.freeze
 
     setting :base_url, DEFAULT_CONFIGURATION[:base_url]
+    setting :client_id, DEFAULT_CONFIGURATION[:client_id]
     setting :client_secret, DEFAULT_CONFIGURATION[:client_secret]
-    setting :token_store, DEFAULT_CONFIGURATION[:token_store]
-    setting :access_token, DEFAULT_CONFIGURATION[:access_token] do |value|
-      next if value.nil? # nil is a valid unassigned value
-
-      invalid_access_token_format!(value) unless value.is_a?(String)
-      config.token_store = { default: value }
-      value
-    end
-    setting :access_tokens, DEFAULT_CONFIGURATION[:access_tokens] do |value|
-      next if value.nil? # nil is a valid unassigned value
-
-      invalid_access_tokens_format!(value) unless value.is_a?(Hash) || value.is_a?(Array)
-      config.token_store = value.is_a?(Hash) ? value : { default: value }
-      value
-    end
+    setting :storage, DEFAULT_CONFIGURATION[:storage]
```

Comment:
I also realised that we should just alias `token_store = store` to be == `token_stores[:default] = store`, that way we can treat the stores the same throughout our code ...